src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,350 @@
+  --pf-c-sequence__number--hover--FontSize: var(--pf-global--FontSize--xl);
+  --pf-c-sequence__number--active--FontSize: var(--pf-global--FontSize--xl);
+  --pf-c-sequence__number--focus--FontSize: var(--pf-global--FontSize--xl);
+  --pf-c-sequence__number--m-current--FontSize: var(--pf-global--FontSize--xl);
+
+  // Number Border
+  --pf-c-sequence__number--BorderColor: var(--pf-global--Color--200);
+  --pf-c-sequence__number--BorderRadius: var(--pf-global--BorderRadius--lg);
+  --pf-c-sequence__number--BorderWidth: var(--pf-global--BorderWidth--md);
+  --pf-c-sequence__number--hover--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__number--active--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__number--focus--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__number--m-current--BorderColor: var(--pf-global--primary-color--100);
+
+  // Content
+  --pf-c-sequence__content--MinHeight: var(--pf-global--spacer--xl);
+
+  // content Title
+  --pf-c-sequence__content-title--MaxWidth: #{pf-size-prem(100px)};
+
+  // Content Title Color
+  --pf-c-sequence__content-title--Color: var(--pf-global--Color--100);
+  --pf-c-sequence__content-title--hover--Color: var(--pf-global--primary-color--100);
+  --pf-c-sequence__content-title--active--Color: var(--pf-global--primary-color--100);
+  --pf-c-sequence__content-title--focus--Color: var(--pf-global--primary-color--100);
+  --pf-c-sequence__content-title--m-current--Color: var(--pf-global--primary-color--100);
+
+  // Content Title Font
+  --pf-c-sequence__content-title--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-sequence__content-title--FontWeight: var(--pf-global--FontWeight--reguar);
+  --pf-c-sequence__content-title--active--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-sequence__content-title--focus--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-sequence__content-title--m-current--FontWeight: var(--pf-global--FontWeight--semi-bold);
+
+  // Content Status
+  --pf-c-sequence__content-status--MaxWidth: #{pf-size-prem(100px)};
+
+  // Content Status Color
+  --pf-c-sequence__content-status--Color: var(--pf-global--Color--200);
+
+  // Content Status Font
+  --pf-c-sequence__content-status--FontSize: var(--pf-global--FontSize--xs);
+  --pf-c-sequence__content-status--FontWeight: var(--pf-global--FontWeight--regular);
+
+  position: relative;
+  display: flex;
+  justify-content: center;
+  height: var(--pf-c-sequence--Height);
+  overflow-y: hidden;
+
+  // remove default border styling
+  button {
+    padding: 0;
+    background-color: transparent;
+    border-top: 0;
+    border-bottom: 0;
+
+    &:focus {
+      outline: none;
+    }
+  }
+
+  button::-moz-focus-inner {
+    border: 0;
+  }
+
+  // remove scrollbar works on Chrome and Safari
+  ::-webkit-scrollbar {
+    display: none;
+    width: 0;
+  }
+}
+
+.pf-c-sequence__control {
```

Comment:
I'm not sure we need lines 143-146. The padding is removed above on line 120 and top and bottom aren't doing anything. Line 120, doesn't seem to doing anything either unless this is browser specific.

Reply:
yeah those aren't affected by different browsers so I can remove